src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationTypes/MS/Internal/Automation/AutomationIdentifierConstants.cs

```diff
@@ -270,7 +278,9 @@ internal enum Properties
             OutlineThickness,
             CenterPoint,
             Rotatation,
-            Size
+            Size,
+            IsDialog,
```

Comment:
IsDialog should come after HeadingLevel.  It was added in RS5, HeadingLevel in RS4.

Reply:
Also, the enum value has to match UIA's property value.  Check the Windows SDK, to make sure.

Reply:
> IsDialog should come after HeadingLevel. It was added in RS5, HeadingLevel in RS4.

Changed this.



> Also, the enum value has to match UIA's property value. Check the Windows SDK, to make sure.

Where should I put the correct values. Directly in the enum?

Reply:
I added the missing properties between size and headinglevel to the enum.